LetsDoStuff.Domain/Models/Activity.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text;
+
+namespace LetsDoStuff.Domain.Models
+{
+    public class Activity
+    {
+        [KeyAttribute]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
```

Comment:
It's look like a little overhead. EF Core defines the properties <User>Id and Id is like PK and identity(1,1). See the https://metanit.com/sharp/entityframeworkcore/2.7.php

Reply:
Yes, it is good to know it, but it is ok to write it explicitly so that won't be confusing for beginners

Reply:
I agree